addons/website_slides/models/slide_slide.py

```diff
@@ -31,6 +31,14 @@ class SlidePartnerRelation(models.Model):
     completed = fields.Boolean('Completed')
 
 
+class SlideLink(models.Model):
+    _name = 'slide.external.link'
+    _description = "External URL for a particular slide"
+
+    slide_id = fields.Many2one('slide.slide', required=True)
+    link = fields.Char("External Link", required=True)
```

Comment:
Not sure: what about a title field with a small compute (like already done in link management) to display it a bit nicely in the frontend (title with href, not hardcoded link tout moche) ?

Reply:
adding a name field.